client/components/Contact.scss

```diff
@@ -5,12 +5,11 @@
 
   &__headline {
     display: block;
-    font-family: Roboto;
     font-size: 48px;
     font-weight: bold;
     margin: 40px 0px;
     line-height: 1.17;
-    color: var(--black);
+    color: #000000;
```

Comment:
I think black is the site-wide default text color; this should probably be set on `body`, where it's currently set to another color, instead of here each time.